packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -686,6 +686,7 @@ function Panel({
             onInputRefChanged={onInputRefChanged}
             placeholderText={placeholderText}
             {...listProps}
+            variant={listProps.groupMetadata?.length ? 'horizontal-inset' : 'inset'}
```

Comment:
Should we maybe just do this in list? and have the logic be internal?

Reply:
the list would have to override it's own variant prop and that feels a bit weird to me 🤔, there'd be no way for the consumer to override it